### PR TITLE
Virgo merge + little code clean up

### PR DESCRIPTION
**Pull request by drexample**
Might be buggy, needs testing

### CODE REVIEW

File: code/game/mecha/equipment/weapons/weapons.dm [hunk 1]
```diff
@@ -159,61 +270,97 @@
...
 	projectile = /obj/item/projectile/bullet/pistol/medium
 	fire_sound = 'sound/weapons/machinegun.ogg'
-	projectiles = 300
+	projectiles = 30 //10 bursts, matching the Scattershot's 10. Also, conveniently, doesn't eat your powercell when reloading like 300 bullets does.
 	projectiles_per_shot = 3
 	deviation = 0.3
 	projectile_energy_cost = 20
 	fire_cooldown = 2
 
-/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/silenced //AEIOU edit
```
Comment (EvilJackCarver) [inline code/game/mecha/equipment/weapons/weapons.dm]: AEIOU edit being removed. Presumably mime gun for a mech.

File: code/game/mecha/equipment/weapons/weapons.dm [hunk 2]
```diff
@@ -280,40 +453,64 @@
...
+	origin_tech = list(TECH_COMBAT= 5, TECH_MATERIAL = 5, TECH_ILLEGAL = 3)
+
+/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/missile_rack/grenade/clusterbang/limited/get_equip_info()//Limited version of the clusterbang launcher that can't reload
 	return "<span style=\"color:[equip_ready?"#0f0":"#f00"];\">*</span>&nbsp;[chassis.selected==src?"<b>":"<a href='?src=\ref[chassis];select_equip=\ref[src]'>"][src.name][chassis.selected==src?"</b>":"</a>"]\[[src.projectiles]\]"
 
-/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/missile_rack/flashbang/clusterbang/limited/rearm()
+/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/missile_rack/grenade/clusterbang/limited/rearm()
 	return//Extra bit of security
 
-//AEIOU EDIT START
```
Comment (EvilJackCarver) [inline code/game/mecha/equipment/weapons/weapons.dm]: Removal of AEIOU code

File: code/game/mecha/mecha.dm [hunk 3]
```diff
@@ -39,7 +40,13 @@
 	var/lights = 0
 	var/lights_power = 6
 	var/force = 0
-	var/silent_step = 0 //Does the mech make noise walking
+
+
+
+	//strafing and actions - AEIOU edit Or addition
```
Comment (EvilJackCarver) [inline code/game/mecha/mecha.dm]: Alters AEIOU code re: strafing many times in this file.

File: code/game/sound.dm [hunk 4]
```diff
@@ -132,7 +132,9 @@
 			if ("mechstep") soundin = pick('sound/mecha/mechstep1.ogg', 'sound/mecha/mechstep2.ogg')
 			if ("geiger") soundin = pick('sound/items/geiger1.ogg', 'sound/items/geiger2.ogg', 'sound/items/geiger3.ogg', 'sound/items/geiger4.ogg', 'sound/items/geiger5.ogg')
 			if ("geiger_weak") soundin = pick('sound/items/geiger_weak1.ogg', 'sound/items/geiger_weak2.ogg', 'sound/items/geiger_weak3.ogg', 'sound/items/geiger_weak4.ogg')
-			if ("keyboard_sound") soundin = pick('sound/effects/keyboard/keyboard1.ogg','sound/effects/keyboard/keyboard2.ogg','sound/effects/keyboard/keyboard3.ogg', 'sound/effects/keyboard/keyboard4.ogg')//AEIOU EDIT
```
Comment (EvilJackCarver) [inline code/game/sound.dm]: AEIOU addition removal

Comment (drexample) [reply code/game/sound.dm]: Unused

**Pull request merged**